src/content/app/species-selector/hooks/useSpeciesSelectorAnalytics.ts

```diff
@@ -18,23 +18,31 @@ import { useSelector } from 'react-redux';
 import analyticsTracking from 'src/services/analytics-service';
 
 import { getSpeciesAnalyticsName } from 'src/content/app/species-selector/speciesSelectorHelper';
-
 import { getCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
 
-import {
+import { AppName } from 'src/global/globalConfig';
+import type {
   PopularSpecies,
   SearchMatch
 } from 'src/content/app/species-selector/types/species-search';
-import { CurrentItem } from 'src/content/app/species-selector/state/speciesSelectorSlice';
+import { type CurrentItem } from 'src/content/app/species-selector/state/speciesSelectorSlice';
+import { type AnalyticsOptions } from 'src/analyticsHelper';
```

Comment:
here and below `import { type  }` -> `import type`?

`import { type  }` makes sense if you are also importing values from the same module; but if you are only importing types, then `import type` is more idiomatic